Fixed the issue with restore/import submenu

VerevkinAlexander opened the pull request:
<h2>General</h2>
Closes #166 
The issue was not the absence of translation, but the fact that Global Settings opened when you clicked on the Restore/Import menu item. This is now fixed. But Restore/Import menu item should probably only appear in Local Wallet Mode. Now in Cloud Wallet Mode this menu item displays only    in the mobile version.

<h2>General</h2>
Please make sure the following is done:

- Pull request is onto develop
- If you haven't already, have a look at
  - https://github.com/bitshares/bitshares-ui/blob/develop/CODE_OF_CONDUCT.md
  - https://github.com/bitshares/bitshares-ui/blob/develop/CONTRIBUTING.md

<h2>Code Preparation</h2>

_Please review all your changes one last time before committing_

- [ ] Check for unused code
- [ ] No unrelated changes are included
- [ ] None of the changed files are reformatting only
- [ ] Code is self explanatory or documented
- [ ] All written text is properly translated (english language)

<h2>Testing</h2>

_The branch has been tested on the following browsers (desktop and mobile view)_

- [ ] Chrome 
- [ ] Opera
- [ ] Firefox
- [ ] Safari

<h2>User interface changes</h2>

_Delete this section if there weren't any UI changes. Please make sure you tested your changes in all themes_

- [ ] Dark
- [ ] Light
- [ ] Midnight

_Please provide screenshots/licecap of your changes below_

VerevkinAlexander committed:
Fixed the issue with restore/import submenu

sschiessl-bcp commented:
This solution is not acceptable.

The settings component is probably not recognizing in what submenu it should jump into, and as no submenu is selcted the translation cant be loaded properly. Please investigate

VerevkinAlexander committed:
Removed the Restore menu item from the cloudWallet

VerevkinAlexander commented:
Done:
1) Restore/Import menu item appeared in the settings menu in mobile version of local wallet by mistake. I have removed it and now the menu looks like shown on screenshots.
2) There was a bug, that leaded to the translation error. I have fixed it. Steps to reproduce:
a) Go to the Login menu and select Login with Key file(local wallet). Close the modal.
b) In the settings menu select any item, that should be visible only in local wallet e.g. Local Wallet.
c) Go to the Login menu again and select Login with Account name(cloud wallet). Close the modal.
d) There is the bug: general settings view opens, but with translation mistake, because there is no Local Wallet window in the cloud wallet.
I've fixed it. Now it redirect to general settings page when user switches to another wallet type.
Cloud wallet menu view:
![menu](https://user-images.githubusercontent.com/43240282/62207661-5a6f6d00-b39d-11e9-9c63-2103c88e2be4.png)
Local wallet menu view:
![menu2](https://user-images.githubusercontent.com/43240282/62207674-5e02f400-b39d-11e9-881c-d8b0efe5962b.png)
Restore window view:
![restore view](https://user-images.githubusercontent.com/43240282/62207675-5fccb780-b39d-11e9-8698-ce7c0a3f8d12.png)

sschiessl-bcp commented:
Accepted
